static/scripts/app-register.js

```diff
@@ -73,24 +84,30 @@ formbtn.addEventListener('click', async (e) => {
         body: jsonData
         });
 
-    switch (datafetch.status) {
-        case 200:
+        const result = await datafetch.json();
+
+        if(datafetch.ok){
             console.log('successful registration');
-            // TODO @mike implement better comms to user
+            let user_succ = document.createElement('small');
+            user_succ.classList.add('big-user-succ');
+            user_succ.innerText = `You Successfully Register!`;
+            // remove err if already exist
+            if(inputsContainer.firstChild.classList?.contains('big-user-succ')
+             || inputsContainer.firstChild.classList?.contains('big-user-err')){
+                inputsContainer.firstChild.remove();
+            }
+            inputsContainer.prepend(user_succ);
+            setTimeout(() => {
+                user_succ.remove();
+            }, 5000);
             window.location.href = '/login';
             return;
-        case 400:
-            console.log('bad request, try again');
-            // TODO @mike implement some communication to user
-            return;
-        case 500:
-            console.log('server error')
-            // TODO @mike implement some communication to user
-            return;
-        case 409:
+        }
+            console.log(result.message)
```

Comment:
indentation looks off

Reply:
nah it's fine

Reply:
It is off, but up to you whether it needs to be fixed, closing this thread